src/js/core/methods/CardanoGetAddress.js

```diff
@@ -162,11 +163,71 @@ export default class CardanoGetAddress extends AbstractMethod {
         return response.message;
     }
 
+    _ensureFirmwareSupportsBatch(batch: Params) {
```

Comment:
Why is it called batch?

Reply:
The `bundle` & `batch` naming seems to be a Trezor convention, see eg. `src/js/core/methods/GetAddress.js` for Bitcoin.